python/sglang/srt/layers/layernorm.py

```diff
@@ -19,93 +19,101 @@
 
 import torch
 import torch.nn as nn
-from flashinfer.norm import (
-    fused_add_rmsnorm,
-    gemma_fused_add_rmsnorm,
-    gemma_rmsnorm,
-    rmsnorm,
-)
 from vllm.model_executor.custom_op import CustomOp
-
-
-class RMSNorm(CustomOp):
```

Comment:
do the same things as `activation.py`

Reply:
`layernorm.py`: things are a bit different -
`vllm._custom_ops` has not defined `gemma_fused_add_rmsnorm`, `gemma_rmsnorm` equivalent, and `rms_norm` API is different.
So can we keep its current change for now?

Reply:
I want to avoid a big indentation for class definition. Can you try this?
I found this is more general, please apply the same to `activation.py` as well.

```python
class RMSNorm(CustomOp):
    def __init__(
        self,
        hidden_size: int,
        eps: float = 1e-6,
    ) -> None:
        super().__init__()
        self.weight = nn.Parameter(torch.ones(hidden_size))
        self.variance_epsilon = eps

    def forward_cuda(
        self,
        x: torch.Tensor,
        residual: Optional[torch.Tensor] = None,
    ) -> Union[torch.Tensor, Tuple[torch.Tensor, torch.Tensor]]:
        ....

    def forward_native(
        self,
        x: torch.Tensor,
        residual: Optional[torch.Tensor] = None,
    ) -> Union[torch.Tensor, Tuple[torch.Tensor, torch.Tensor]]:
        ....

if is_hip():
    logger.info("FlashInfer is not available on AMD GPUs. Fallback to other kernel libraries.")
    from vllm.model_executor.layers.layernorm import GemmaRMSNorm, RMSNorm
    RMSNorm.forward_cuda = RMSNorm.forward_cuda
```

Reply:
This reorder works! Let me make the change :)